WebApplication1/WebApplication1/Controllers/DrinkController.cs

```diff
@@ -21,7 +21,9 @@ public DrinkController(IDrinkRepostory drinkRepostory, ICategoryRepostory catego
             this.categoryRepostory = categoryRepostory;
         }
 
-        public IActionResult List(string category)
```

Comment:
try to not write any code in the controller and move all this logic to another class which will be responsible for applying the business logic 
https://www.c-sharpcorner.com/article/three-tier-architecture-in-asp-net-core-6-web-api/